src/Sarif.Viewer.VisualStudio.Core/Services/DataService.cs

```diff
@@ -108,6 +112,8 @@ private async System.Threading.Tasks.Task<int> SendEnhancedResultDataAsync(Sarif
                 this.sarifErrorListEventSelectionService.SelectedItem = items[0];
 
                 items[0].Locations?.FirstOrDefault()?.NavigateTo(usePreviewPane: false, moveFocusToCaretLocation: true);
+
+                this.keyEventTelemetry.TrackEvent("DisplayKeyEventData");
```

Comment:
>DisplayKeyEventData [](http://example.com/codeflow?start=51&length=19)

Should we put these in some central location?

Reply:
updated